### PR TITLE
feature(unlock-app): ux improvements for clarity around Guild Hook

### PR DESCRIPTION
**Pull request by julien51**
# Description

This makes the UI more clear as well as add a polling check.

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/hooks/useDataForGuild.ts [hunk 1]
```diff
@@ -25,7 +25,14 @@ export function useDataForGuild({
...
-    return getDataForGuild(network, lockAddress, recipients)
-  })
+  return useQuery(
+    ['getLockSettings', lockAddress, network],
+    async () => {
+      return getDataForGuild(network, lockAddress, recipients)
+    },
+    {
+      refetchIntervalInBackground: true,
+      refetchInterval: 3000, // Every 3 seconds?
```
Comment (searchableguy) [inline unlock-app/src/hooks/useDataForGuild.ts:35]: It's excessive. What's the use case? If it is to refresh based on changing tabs or something similar, use a more specific event like refreshOnFocus or similar.

Comment (julien51) [reply unlock-app/src/hooks/useDataForGuild.ts:35]: > It's excessive. What's the use case? If it is to refresh based on changing tabs or something similar, use a more specific event like refreshOnFocus or similar.

I will look at `refreshOnFocus`!

Comment (julien51) [reply unlock-app/src/hooks/useDataForGuild.ts:35]: ho it looks like this is the default!

Comment (julien51) [inline unlock-app/src/hooks/useDataForGuild.ts:35]:
```suggestion
```

File: unlock-app/src/hooks/useDataForGuild.ts [hunk 2]
```diff
@@ -25,7 +25,12 @@ export function useDataForGuild({
...
 }: UseDataForGuildProps) {
-  return useQuery(['getLockSettings', lockAddress, network], async () => {
-    return getDataForGuild(network, lockAddress, recipients)
-  })
+  return useQuery(
+    ['getLockSettings', lockAddress, network],
+    async () => {
+      return getDataForGuild(network, lockAddress, recipients)
+    },
+    {
```
Comment (julien51) [inline unlock-app/src/hooks/useDataForGuild.ts:33]:
```suggestion
```

**Pull request merged**